model/src/w3iogomd.F90

```diff
@@ -1631,12 +1602,16 @@ SUBROUTINE W3OUTG ( A, FLPART, FLOUTG, FLOUTG2 )
           TPMS(JSEA) = TPI/SIG(IK)
         END IF
 
-#ifdef W3_CESMCOUPLED
-        ! Get surface layer depth
-        IX    = MAPSF(ISEA,1)
-        IY    = MAPSF(ISEA,2)
-        HSL   = HML(IX,IY)/5.     ! depth over which SD is averaged
-#endif
+         IF (LMPENABLED) then
+            IF (HSLMODE.EQ.0) then
+              LHSL = 10.0 ! a constant value for testing purposes
+            ELSE
+              ! Get surface layer depth from coupler
+              IX    = MAPSF(ISEA,1)
+              IY    = MAPSF(ISEA,2)
+              LHSL  = HSL(IX,IY)      ! depth over which SD is averaged
+              END IF
```

Comment:
Please indent this code correctly (also LN 1657, 1662, 1681, 1964, 2011 below).

Reply:
Sure, let look into it.

Reply:
I can't commit to this branch directly, but I'll get a branch in the works to raise a sub-pr.  The timeline is Monday.  @DeniseWorthen @JessicaMeixner-NOAA

Reply:
@DeniseWorthen, I've addressed each of these lines in this PR to ESCOMP: https://github.com/ESCOMP/WW3/pull/20, which is now ready for review.